### DISCUSSION TITLE
(Feature Request): Azure Monitor & template variable: Ability to query Azure Resource Graph

### DISCUSSION THREAD

**Discussion by wengelbrecht-grafana** [grafana/grafana#44157] [Dashboards, Panels & Visualizations]
I'm working on a project where we would like to create Dashboard template variables from Azure Monitor plugin based on the Azure Resource Graph capability. Currently when using the Azure Monitor plugin, you can only query "Logs" or "Grafana function" in template variables: 
![image](https://user-images.githubusercontent.com/83230950/149956317-49e3bd77-d811-4619-8579-63250ac5e7c4.png)
It would be really ideal to also have the option to pick Azure Resource Graph from this list, and then we can query the tags we're using in Grafana. 

A side note, you can query Azure Resource Graph in the Panel editor already, you just can't do it from template variables:
![image](https://user-images.githubusercontent.com/83230950/149956479-dacf22a2-0ef3-4da7-9c91-6a41f380c52b.png)

**Comment by sarahzinger** [comment]
we actually have brainstormed out some plans for this :) https://github.com/grafana/grafana/discussions/39970

**Comment by andresmgot** [comment]
Created an issue for this: https://github.com/grafana/grafana/issues/50233

Please use the issue if further discussion is necessary!
